### ISSUE TITLE
Make the awful Hissho Dark Matter quest give the Hissho Hunter blueprint

### ISSUE THREAD

**Issue by captaincobbs** [wakystuf/ESG-Mod#994]
Replace Behemoth tactic, it's useless no matter the numbers.

**Comment by wakystuf** [comment]
@captaincobbs Am I correct that this is a reference to the truly awful Berserker tactic (PlayOffensiveShortDamageBoost, unlocked by TechnologyPlayDeckQuestHisshos05, which is in turn attached to TechnologyDefinitionQuestHisshos05)? If so, then @FluffiestPrincess , the goal is to tweak that empire improvement reward to instead unlock the Hissho hunter blueprint (similar to how Cobbs made some of the other quests unlock the Nakalim attacker and Vodyani carrier)

**Comment by captaincobbs** [comment]
Yes, one stipulation however, if the Ship Design Rewards setting is toggled off OR Supremacy is not enabled then it should use the vanilla quest that gives the awful tactic.
